src/AppInstallerCLICore/Command.cpp

```diff
@@ -183,6 +200,37 @@ namespace AppInstaller::CLI
             std::endl <<
             std::endl;
 
+        if (!commandAliases.empty())
+        {
+            infoOut << Resource::String::AvailableCommandAliases << std::endl << "  "_liv;
+            
+            size_t i = 1;
+            for (const auto& commandAlias : commandAliases)
+            {
+                infoOut << Execution::HelpCommandEmphasis << commandAlias;
+                /*
+                *  Print up to 7 command aliases per line, comma separated for readability
+                *  This is intended to be future-looking so that in case common typos are
+                *  added to command aliases as scoop/chocolatey do, the list does not become
+                *  extremely long to the point where it makes the help page less usable.
+                */

```

Comment:
For your reference @denelon -
![image](https://user-images.githubusercontent.com/12611259/181798253-68576bf5-d1b0-45a9-83e2-454d38dfcfb4.png)